bin/run-browser-test-env

```diff
@@ -12,45 +12,73 @@ if [ -f ".secrets" ]; then
   . .secrets
 fi
 
+ci="no"
+
 # Get cloud provider
 _setArgs(){
   while [ "${1:-}" != "" ]; do
     case "$1" in
+      "--ci")
+        ci="yes"
+        ;;
+
       "--azure")
-       cloudProvider="azure"
-       emulator="azurite"
-       devServer="http://localhost:10000"
-       export STORAGE_SERVICE_NAME="azure-blob"
-       ;;
+        readonly CLOUD_PROVIDER="azure"
+        readonly EMULATOR="azurite"
+        readonly SERVER_URL="http://localhost:10000"
+        export STORAGE_SERVICE_NAME="azure-blob"
+        ;;
+
       "--aws")
-       cloudProvider="aws"
-       emulator="localstack"
-       devServer="http://localhost:6645"
-       export STORAGE_SERVICE_NAME="s3"
+        readonly CLOUD_PROVIDER="aws"
+        readonly EMULATOR="localstack"
+        readonly SERVER_URL="http://localhost:6645"
+        export STORAGE_SERVICE_NAME="s3"
        ;;
     esac
+
     shift
   done
 }
 
 _setArgs "$@"
 
 # default to AWS for now
-if [ -z "$cloudProvider" ]; then
-  cloudProvider="aws" && emulator="localstack" && devServer="http://localhost:6645"
+if [ -z "${CLOUD_PROVIDER}" ]; then
+  readonly CLOUD_PROVIDER="aws"
+  readonly EMULATOR="localstack"
+  readonly SERVER_URL="http://localhost:6645"
+
   export STORAGE_SERVICE_NAME="s3"
 fi
 
+compose_command="docker-compose --profile ${CLOUD_PROVIDER}"
+compose_command="${compose_command} -f browser-test/browser-test-compose.yml"
+
+if [[ "${ci}" = "no" ]]; then
+  compose_command="${compose_command} -f browser-test/browser-test-compose.dev.yml"
+fi
+
+compose_command="${compose_command} up"
+
+readonly START_EMULATOR_COMMAND="${compose_command} -d ${EMULATOR}"
+echo "Running docker compose to start the cloud emulator:"
+echo "${START_EMULATOR_COMMAND}"
+
 # start emulator
-docker-compose --profile $cloudProvider \
-  -f browser-test/browser-test-compose.yml \
-  -f browser-test/browser-test-compose.dev.yml \
-  up -d $emulator
+eval "${START_EMULATOR_COMMAND}"
+
+# wait until the emulator is running
+"bin/${EMULATOR}/wait" "${SERVER_URL}"
+
+if [[ "${ci}" = "yes" ]]; then
+  compose_command="${compose_command} > .dockerlogs 2>&1 &"
+fi
 
-# wait until emulator running
-bin/$emulator/wait $devServer
+echo "Running docker compose to start the CiviForm environment:"
+echo "${compose_command}"
 
 # start everything else
```

Comment:
Move this to L74?

Reply:
The comment?

Reply:
yeah, I read it as a section comment and the section starts further above now. optional.

Reply:
Ah, leaving out for now since it feels more like it's specific to the eval statement. LMK if you'd prefer I move it or add a comment for the stuff above